src/stores/index.ts

```diff
@@ -226,7 +225,7 @@ export const useStore = defineStore("store", {
           this.calculs.resultats._id === this.simulationId
       )
     },
-    situation(): Situation {
+    situation() {
```

Comment:
Import de /lib/types/situations.d.ts
```suggestion
     situation(): Situation | Record<string, never> {
```